### PR TITLE
Remedial work completed

### PR DESCRIPTION
**Pull request by shanel262**
#6 closed.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+node_modules
+*.log
+.DS_Store
+
+dist
```
Comment (ckiss) [inline .eslintignore]: you don't need dist and client here

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+*.pid
+*.gz
+
+pids
+logs
+results
+
 npm-debug.log
-.DS_Store
-config.mine.js
```
Comment (ckiss) [inline .gitignore]: you can add back config.mine.js

File: example/twitter-auth-example.js [hunk 3]
```diff
@@ -1,35 +1,35 @@
 /* Copyright (c) 2012-2013 Richard Rodger, MIT License */
-"use strict";
+'use strict'
 
 
-var http = require('http')
+var HTTP = require('http')
```
Comment (ckiss) [inline example/twitter-auth-example.js]: not HTTP, just Http

File: example/twitter-auth-example.js [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
 
-var express = require('express')
-var argv    = require('optimist').argv
+var EXPRESS = require('express')
```
Comment (ckiss) [inline example/twitter-auth-example.js]: same as above

File: package.json [hunk 5]
```diff
@@ -4,7 +4,9 @@
   "description": "twitter auth plugin for seneca-auth",
   "main": "twitter-auth.js",
   "scripts": {
-    "test": "mocha test --reporter spec"
+    "test": "mocha test --reporter spec",
+    "lint": "lab test -dL",
+    "annotate": "docco local-auth.js -o doc"
```
Comment (ckiss) [inline package.json]: change filename to twitter-auth.js :)

**Comment by ckiss** [comment]
@shanel262 looks good, just a few small comments that needs to be fixed before merge. don't forget to add yourself as contributor :)

File: .eslintignore [hunk 6]
```diff
@@ -0,0 +1,3 @@
+node_modules
+*.log
+.DS_Store
```
Comment (mihaidma) [inline .eslintignore]: no newline

File: .gitignore [hunk 7]
```diff
@@ -1,5 +1,18 @@
...
+pids
+logs
+results
+
 npm-debug.log
+node_modules
+*~
 .DS_Store
-config.mine.js
+config.mine.js
```
Comment (mihaidma) [inline .gitignore]: no newline

File: LICENSE [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
Comment (mihaidma) [inline LICENSE]: no newline at end of file

File: docs/public/stylesheets/normalize.css [hunk 9]
```diff
@@ -0,0 +1,375 @@
...
+   ========================================================================== */
+
+/*
+ * Remove most spacing between table cells.
+ */
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
```
Comment (mihaidma) [inline docs/public/stylesheets/normalize.css]: no newline at end of file

**Pull request merged**